libs/route-config/src/lib/route-data/route-data.directive.spec.ts

```diff
@@ -0,0 +1,6 @@
+describe('RouteDataDirective', () => {
+  it('should create an instance', () => {
+    // const directive = new RouteDataDirective();
```

Comment:
I believe you forgot to write tests for the `route-data.directive` :)

Reply:
done